src/pages/dialog/Dialog.module.css

```diff
@@ -0,0 +1,32 @@
+body {
+	background-color: #f1f3f4;
+}
+/* .Container {
+	
+} */
+
+.Dialog {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	margin-top: 10px;
+	margin-bottom: 10px;
+	height: 95vh;
+}
+
+/*  WE HAVE BUG HERE BELOW      !!!!!!!!!!!!!!!!!!!!! */
+/*  WE HAVE BUG HERE BELOW      !!!!!!!!!!!!!!!!!!!!! */
+/*  WE HAVE BUG HERE BELOW      !!!!!!!!!!!!!!!!!!!!! */
+/*  WE HAVE BUG HERE BELOW      !!!!!!!!!!!!!!!!!!!!! */
+/* overflow property doesnt works perfectly(((( */
+.DialogField {
```

Comment:
For fix design change this to
display: flex;
flex-direction: column;
background-color: white;
padding: 20px;
overflow-y: scroll;
max-height: infinite;